setup.py

```diff
@@ -177,7 +177,7 @@ def build_extension(self, ext):
     name='depthai',
     version=__version__,
     author='Martin Peterlin',
```

Comment:
Change to the name of the company "Luxonis" like in depthai-sdk, or any other package on pypi?
https://pypi.org/project/tensorflow/#data

You can add "maintainer" as a person with private email.